scripts/run-ginkgo-integration-suite.sh

```diff
@@ -9,7 +9,7 @@ set -e
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 GINKGO_TEST_BUILD="$SCRIPT_DIR/../test/build"
-SKIP_MAKE_TEST_BINARIES=1
+: "${SKIP_MAKE_TEST_BINARIES:=}"
```

Comment:
What does `:` in the front do? Does it nullify the variable `SKIP_MAKE_TEST_BINARIES` ?

Reply:
That defines the env variable if it is not already defined (not already passed to the script) and lets you pass a default value on the other side of `:=`, which in this case is `nil`.